### DISCUSSION TITLE
how to update ezinsar to new version?

### DISCUSSION THREAD

**Discussion by Rick0219** [alexisInSAR/EZ-InSAR#85] [Help]
Hi Sir.,
I want to update the old version ezinsar to a new one,it would be grateful to tell me how to update it.maybe just delete the old ezinsar bin and download the new one from github?

**Comment by Rick0219** [comment]
@alexisInSAR

**Reply by alexisInSAR** [reply]
Hi, 

Sorry for my late reply. Just download the new version and replace the files. 

Alexis
